packages/zosfiles/src/api/methods/download/Download.ts

```diff
@@ -87,10 +87,14 @@ export class Download {
             // Note that the "extension" options do not affect the destination if the "file" options were provided
             const destination = (() => {
 
-                const cwd = options.cwd;
+                let lcd = options.lcd;
+
+                if (lcd && lcd[lcd.length - 1] !== '/') {
+                    lcd += '/';
+                }
```

Comment:
Could we use `path` functions here?
What do you think about `let lcd = options.lcd ? path.normalize(path.join(options.lcd, "/")); : null` ?